content/docs/0.8.x/continuous_delivery/deployment_helm/index.md

```diff
@@ -75,6 +75,54 @@ carts-primary   1/1     1            1            4m   carts        docker.io/ke
 carts           0/0     0            0            1d   carts        docker.io/keptnexamples/carts:0.11.2   app=carts
 ```
 
+
+## User Managed deployments (experimental)
+
+:warning: *Note, that this feature is currently marked as experimental. Hence, expect various enhancements and/or changes in
+future releases of Keptn.*
+
+
+If the deployment strategy of a stage in the [shipyard](https://github.com/keptn/examples/blob/0.8.0/onboarding-carts/shipyard.yaml)
+is configured as `deploymentstrategy: user_managed`, the provided Helm chart will be deployed without any modification and
+applied as it is.
+
+Assuming that you've created a project from a shipyard file containing a stage with `deploymentstrategy: user_managed`, you need to: 
+
+(1) create a service in your stage:
+```
+keptn create service <service-name> --project <project-name>
+```
+(2) add your desired Helm chart to each stage:
+```
+keptn add-resource --project=<project-name> --service=<service-name> --all-stages --resource=<your-helm-chart.tgz> --resourceUri=helm/<service-name>.tgz
+```
+(3) send an event to trigger the delivery of your service:
+```
+keptn send event --file=./delivery.json 
+```
+where the content of `delivery.json` looks something like:
+
+```
+{
+  "contenttype": "application/json",
+  "data": {
+    "project": "<project-name",
+    "service": "<service-name>",
+    "stage": "<stage-name>" 
+  },
+  "source": "keptn-cli",
+  "specversion": "1.0",
+  "type": "sh.keptn.event.<stage-name>.delivery.triggered"
+}
+```
+
+**Limitations:**
+- Currently, when using this deployment strategy, you cannot use the `keptn trigger delivery` command as you would
+normally do. Hence, you need to send an event using `keptn send event` like described above.
+- Modifications to the Helm chart via keptn configuration changes are currently not possible.
```

Comment:
Formatting is weird in the rendered version:

![image](https://user-images.githubusercontent.com/56065213/109836152-2d073700-7c44-11eb-82ea-0cc9d085496d.png)